src/components/[guild]/AccessHub/components/RemovePlatformMenuItem.tsx

```diff
@@ -13,17 +13,15 @@ type Props = {
 
 const RemovePlatformMenuItem = ({ platformGuildId }: Props): JSX.Element => {
   const { isOpen, onOpen, onClose } = useDisclosure()
-
+  const color = useColorModeValue("red.600", "red.300")
   const { guildPlatforms } = useGuild()
-  const guildPlatform = guildPlatforms.find(
+  const guildPlatform = guildPlatforms?.find(
     (gp) => gp.platformGuildId === platformGuildId
   )
-
-  const { isPlatform } = rewards[PlatformType[guildPlatform?.platformId]] ?? {}
-
   const { onSubmit, isLoading } = useRemoveGuildPlatform(guildPlatform?.id)
-
-  const color = useColorModeValue("red.600", "red.300")
+  if (guildPlatform === undefined)
+    throw new Error(`Unmatched guild platform ID ${platformGuildId}`)
```

Comment:
This would cause a client-side error, and we'd like to avoid such things, so I think we should just return null instead, what do you think? (I believe this case shouldn't even happen actually, but I know TS yells at it). Alternatively, we could introduce new error boundaries, but I think that'd be too much effort for this case.